lib/jnpr/junos/rpcmeta.py

```diff
@@ -295,41 +295,38 @@ def _exec_rpc(*vargs, **kvargs):
             # create the rpc as XML command
             rpc = etree.Element(rpc_cmd)
 
+            # Gather decorator keywords into dec_args and remove from kvargs
+            dec_arg_keywords = ['dev_timeout', 'normalize', 'ignore_warning']
+            dec_args = {}
+            for keyword in dec_arg_keywords:
+                if keyword in kvargs:
+                    dec_args[keyword] = kvargs.pop(keyword)
```

Comment:
As we are supporting Python 2.6, else dict comprehension would have been a good option here.